docs/Maths.md

```diff
@@ -226,6 +226,107 @@ Result:
 ![](plotly_svg.png)
 
 
+## Using Pygal ##
+[Pygal](https://www.pygal.org/en/stable/) is a graph plotting library using python. You can install Pygal using ` $ pip install pygal` command. 
+
+`fpdf2` is able to embed the graph and charts that are generated using `Pygal` library. The following ways explain how to embed `Pygal` charts into `fpdf2` library.
```

Comment:
Could you please add a sentence there explaining why we cannot embed graphs as SVG directly,
as we discussed in https://github.com/PyFPDF/fpdf2/issues/652#issuecomment-1501601977? 😊

Reply:
OK @Lucas-C . I will add and commit this ASAP.